### PR TITLE
API-1783: Rebase 1.30

### PR DESCRIPTION
**Pull request by soltysh**
Replaces https://github.com/openshift/origin/pull/28828
/assign @dinhxuanvu

### CODE REVIEW

File: test/extended/operators/server_side_apply_examples.go [hunk 1]
```diff
@@ -101,15 +100,22 @@ var _ = g.Describe("[sig-apimachinery]", func() {
 				g.Skip("only SelfManagedHA and SingleNode have mutable FeatureGates")
 			}
 
+			condTrue := metav1.ConditionTrue
+			now := metav1.Now()
+			type1 := "FristType"
```
Comment (dinhxuanvu) [inline test/extended/operators/server_side_apply_examples.go:105]: Typo. Should be `FirstType`.

**Comment by dinhxuanvu** [comment]
Holding this PR for now as we are reverting 1.30 rebase on o/k side.
/hold

**Comment by dinhxuanvu** [comment]
/payload-job periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-ovn-serial

**Comment by soltysh** [comment]
/payload-job periodic-ci-openshift-release-master-nightly-4.17-e2e-metal-ipi-upgrade-ovn-ipv6

**Comment by dinhxuanvu** [comment]
The issue with the `metal-ipi-upgrade-ovn-ipv6` payload job seems to be an image issue as the ocp image was built and pushed to CI registry instead of openshift quay registry. As a result, the ocp image isn't pullable on some test and failed. This issue doesn't seem to be related the rebase itself.

**Pull request merged**